hw/top_earlgrey/top_earlgrey.core

```diff
@@ -44,6 +44,17 @@ filesets:
       - rtl/padctl.sv
       - rtl/autogen/top_earlgrey.sv
     file_type: systemVerilogSource
+  files_rtl_asic:
+    depend:
+      - lowrisc:prim_asic:clock_gating
+      - lowrisc:prim_asic:clock_mux2
+      - lowrisc:prim_asic:pad_wrapper
+      - lowrisc:prim_asic:ram_1p
+      - lowrisc:prim_asic:ram_2p
+      - lowrisc:prim_asic:rom
+      - lowrisc:prim_asic:flash
+    files:
+    file_type: systemVerilogSource
```

Comment:
These dependencies should be added to `hw/ip/prim/prim_*` and can be removed here.

Reply:
So as @tjaychen mentioned, these are omitted on purpose, since they contain tech-specific details which we are not allowed to expose. 

I can add stubs for these such that other partners can adopt them. I can also lump them together in a `prim_asic.core` to reduce clutter, but they would still have to be listed as a separate dependency here. WDYT?

Reply:
i.e. `lowrisc:prim_asic:all`